lib/thumbs/pull_request_worker.rb

```diff
@@ -81,7 +81,20 @@ def try_merge
         git.checkout(pr.base.ref)
         git.branch(pr_branch).checkout
         debug_message "Trying merge #{@repo}:PR##{@pr.number} \" #{@pr.title}\" #{most_recent_head_sha} onto #{@pr.base.ref} #{most_recent_base_sha}"
-        merge_result = git.merge(most_recent_head_sha)
+
+        if forked_repo_branch_pr?
+          contributor_repo="git://github.com/#{pr.head.repo.full_name}"
+          debug_message("Forked branch pr contributor REPO: #{contributor_repo}")
+          remotes=git.remotes.collect{|r| r.name }
+          unless remotes.include?("contributor")
```

Comment:
This use of `"contributor"` instead of the contributors name is okay as long as either:
1. The remotes added is removed certainly after the merge (using a rescue to ensure that the removal surely happens).
2. The remote is instead named using the contributor's handle which is distinct.